driver/sdk.md

```diff
@@ -311,24 +311,59 @@ change it since it'll execute the driver.
 
 The integration tests will test all the process of the driver from the request
 to the parser to the annotated UAST generation. They work by comparing the
-UAST output with the previously existing one, and will fail if any difference
-is found.
+UAST and native output with the previously existing one, and will fail if any 
+difference is found.
 
 For this to work, the driver developer will need to provide files with source
-code examples in the `tests/` directory. with the `.source` file extension.
-Once this is done, you need to run the integration-test Makefile's target for
-the first time to automatically generate  the `.native` and `.uast` files:
+code examples in the `tests/` directory. with the `.source` file extension. It's
+recommended that you keep the original extension before the `.source` so the
+command below can autodetect the language.
+
+Once this is done, you need to generate the `.native` and `.uast` fixtures with 
+the command:
+
+```
+# Only if you didn't already had a bblfshd server running:
+docker run -d --name bblfshd -p 9432:9432 bblfsh/bblfshd
+# Only if you didn't already had the drivers installed:
+docker exec -it bblfsh bblfshctl driver install --all
+
+bblfsh-sdk-tool fixtures fixtures/*.source
+```
+
+If the command fails with the error `unexpected error: runtime failure: missing 
```

Comment:
I don't think this makes sense. If you already have installed all the drivers you shouldn't see this problem, but if you see this error how can passing the language explicitely help with that?

Reply:
This error happens not only when there is not a driver but also when enry fails to detect the language, usually because it doesn't have the extension (like in the case of the Java fixtures, trough I'll make a PR about them). I'll be more explicit about this.

Reply:
I'd say that this is then a bug in the server. The server shouldn't return this error when `enry` fails to detect the language.

Reply:
Yes. The problem is that enry.GetLanguage() only returns an empty string if it can't detect anything and then the server searchs for a driver called `""` which obviously doesn't work. It should detect the empty string and produce an specific error (I'll make a server PR update the doc).